app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_import.scala.html

```diff
@@ -78,10 +78,20 @@ <h3 class="govuk-heading-m">@messages("inspection_needed_import.arriving.h3")</h
 
     <p class="govuk-body">@Html(messages("inspection_needed_import.arriving.p1"))</p>
     <ul class="govuk-list govuk-list--bullet" >
-        <li><a href="@{appConfig.attendingInlandBorderFacilityUrl}">@messages("inspection_needed_import.arriving.p1.link1")</a></li>
-        <li><a href="@{appConfig.checkDelaysInlandBorderFacilityUrl}">@messages("inspection_needed_import.arriving.p1.link2")</a></li>
+        <li><a href="@{
+            appConfig.attendingInlandBorderFacilityUrl
+        }">@messages("inspection_needed_import.arriving.p1.link1")</a></li>
+        <li><a href="@{
+            appConfig.checkDelaysInlandBorderFacilityUrl
+        }">@messages("inspection_needed_import.arriving.p1.link2")</a></li>
     </ul>
 
+
+    @if(isInbound) {
```

Comment:
I think there's potential for confusion with the boolean `isInbound`. Instead of having this boolean, can we just pass the direction in and check that it's EU to GB (which is admittedly poorly named UK_INBOUND)?

ie. just have a check that `direction == UK_INBOUND` within the view.

Reply:
updated